doc/usage/restructuredtext/directives.rst

```diff
@@ -394,6 +394,8 @@ units as well as normal text.
 
    .. versionadded:: 0.6
 
+   .. versionchanged:: 3.5.0
+      support by the latex builder.
```

Comment:
Should this be 3.5 not 3.5.0? And should I capitalize: "Support", not "support"?

Reply:
Both are okay, but I usually use 3.5 and "Support" (no reason and rules :-p)

Reply:
Ah, I did not noticed this is a changes of `hlist` directive. I don't think this changelog is needed because of this change does not modify the feature of the directive itself.